app/serializers/location_serializer.rb

```diff
@@ -3,5 +3,13 @@
 class LocationSerializer < ActiveModel::Serializer
   type 'locations'
 
-  attributes :id, :key, :title, :location_type, :nomis_agency_id, :can_upload_documents, :disabled_at, :suppliers
```

Comment:
since suppliers always came with locations and now will no longer will be available unless explicitly mentioned, we might need to be explicit to include it everywhere we include the location resource in other serialisers example moves etc.

Reply:
example https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/serializers/move_serializer.rb#L30

Reply:
Having this conversation with Alex currently

Reply:
Also, https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/525#issuecomment-637454876

Reply:
Okay, had a chat with Alex. We're going to add support for the query params and static list as elsewhere.